endpoints/cps.yaml

```diff
@@ -0,0 +1,295 @@
+        "4XX":
+          description: Client error.
+          content:
+            application/json:
+              schema:
+                $ref: "../components/Error.yaml"
+  /image.localize:
+    post:
+      description: Compute a high accuracy global position and orientation for the image.
+      operationId: localizeImage
+      parameters:
+        - in: header
+          name: Fantasmo-Key
+          schema:
+            type: string
+          required: true
+      requestBody:
+        content:
+          multipart/form-data: # Media type
+            schema: # Request payload
+              type: object
+              required:
+                - capturedAt
+                - image
+                - intrinsics
+                - uuid
+              properties: # Request parts
+                capturedAt:
+                  description: Time in seconds since Unix Epoch that this image was captured.
+                  type: number
+                  format: double
+                coordinate:
+                  $ref: "../components/Coordinate.yaml"
+                image:
+                  description: Query image for localization.
+                  type: string
+                  format: binary
+                mapId:
+                  description: ID of the map to query. If no map ID is provided, the `coordinate` is used to perform a map lookup. If neither are provided, an error is returned.
+                  type: string
+                intrinsics:
+                  $ref: "../components/Intrinsics.yaml"
+                uuid:
+                  $ref: "#/components/schemas/Uuid"
+                gravity:
+                  $ref: "#/components/schemas/Gravity"
+                position:
+                  $ref: "../components/Position.yaml"
+                referenceFrame:
+                  $ref: "#/components/schemas/ReferenceFrame"
+                imageResolution:
+                  $ref: "../components/ImageResolution.yaml"
+                rotationSpread:
+                  description: Total rotation spread since the latter of the start of the session and the last anchor set. Computed as the difference between the minimum and maximum angle along principal axes in radians.
+                  type: object
+                  required:
+                    - pitch
+                    - yaw
+                    - roll
+                  properties:
+                    pitch:
+                      description: Dimension of rotation on the up-down axis.
+                      type: number
+                      format: float
+                    yaw:
+                      description: Dimension of rotation on the left-right axis.
+                      type: number
+                      format: float
+                    roll:
+                      description: Dimension of rotation on the front-back axis.
+                      type: number
+                      format: float
+                totalDistance:
```

Comment:
We had decided this would be a vector called `totalTranslation`. Which is correct?

https://linear.app/fantasmo/issue/MW-89/monitor-and-accumulate-device-motion

Reply:
We already have totalDistance in the SDK, so going with that.